packages/protobuf-bench/src/gen/protobuf-es/buf/alpha/audit/v1alpha1/user_pb.ts

```diff
@@ -25,17 +25,17 @@ export enum BufAlphaRegistryV1Alpha1UserState {
   /**
    * @generated from enum value: BUF_ALPHA_REGISTRY_V1_ALPHA1_USER_STATE_UNSPECIFIED = 0;
    */
-  BUF_ALPHA_REGISTRY_V1_ALPHA1_USER_STATE_UNSPECIFIED = 0,
+  UNSPECIFIED = 0,
```

Comment:
A breaking change, but a nice one. We always stripped shared enum value prefixes in the generated code, but the logic got confused with alphanumeric names.